arch/arc/Kconfig

```diff
@@ -105,6 +105,11 @@ endif #ISA_ARCV3
 config FP_FPU_DA
 	bool
 
+config ARC_DSP_COMPLEX
+	bool
+	depends on DSP
+	default y
```

Comment:
Hmm, so we can enable it for all ARC targets. Is it really supported for both EM, ARCv2 HS, ARCv3 HS? 
If no - let's limit it to the exact processor type.

Reply:
fiexed